Tasks/AzureCloudPowerShellDeploymentV1/task.json

```diff
@@ -139,6 +139,14 @@
             "helpMarkDown": "When selected sets the upgrade to a forced upgrade, which could potentially cause loss of local data.",
             "required": false
         },
+        {
+            "name": "CheckForInstancesReady",
+            "type": "boolean",
+            "label": "Check for instances ready",
```

Comment:
imo 'verify role instance status' is a better label. Can you please check with PM on this.